coyote/README.md

```diff
@@ -1,4 +1,5 @@
-# Coyote Analysis Pipeline (canFAM3.1)
+# Coyote Analysis Pipeline (Dog10K Boxer Tasha)*
```

Comment:
Is this correct, coyote README notes dog 10k?

Reply:
Good catch, forgot to revert this change